provisioning/roles/nginx/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Add NGINX signing key
+  apt_key:
+    url: "http://nginx.org/keys/nginx_signing.key"
+
+- name: Add NGINX APT repository
+  apt_repository: repo='deb http://nginx.org/packages/debian/ {{ansible_distribution_release}} nginx' state=present
+
+- name: Add NGINX source APT repository
+  apt_repository: repo='deb-src http://nginx.org/packages/debian/ {{ansible_distribution_release}} nginx' state=present
+
+- name: Ensure NGINX is installed at the latest version
+  apt: name=nginx state=latest
```

Comment:
Euh, après vérification, dans les dépôts Debian, on a NGINX 1.2. (wheezy, si je dis pas de connerie est la dernière version stable de Debian).
La dernière version (according to nginx.org) est NGINX 1.10.

La réponse est juste que je veux probablement profiter de HTTP/2 disponible à partir de NGINX 1.9. Je pense aussi qu'avoir cette version est très pratique et ne coûte quasiment pas grand chose à part ces quelques lignes.